src/System.Collections/tests/Configurations.props

```diff
@@ -2,7 +2,6 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
```

Comment:
Do we not have any new System.Collection APIs that are netcoreapp specific? it is possible but I would have thought we added some and if we did we might still need this configuration.